### ISSUE TITLE
Update Image Size - Command Issue

### ISSUE THREAD

**Issue by jakeanspach** [emmetio/sublime-text-plugin#63]
Looks like it only works for embedded styles now as before the command worked inline as well. I work in email dev and this function is such an asset. Am I missing something?

**Comment by sergeche** [comment]
By inline you mean inside `style=""` attribute or inside `<style>` tag of HTML?

**Comment by jakeanspach** [comment]
In the past when you used the command `ctr+shift+i` or `cmd+shift+u` within the `<img>` tag it would populate the `width=""` and `height=""`.

**Comment by sergeche** [comment]
Keyboard shortcuts are not provided by default since lots of users complained about Emmet actions overriding default actions. You should add them manually: https://github.com/emmetio/sublime-text-plugin#more-actions

**Comment by jakeanspach** [comment]
So I added the shortcut in the keymap settings and it works great in embedded styles like:

```css
<style>
.bg { background-image: url('images/demo.jpg');
</style>
```
but not in the `<img>` tag.

**Comment by sergeche** [comment]
Can you give me example of `<img>` you’re trying?

**Comment by jakeanspach** [comment]
`<img alt="" src="http://mg-email-content-images.s3.amazonaws.com/090320_CFA_385086_SeptemberEmail1_LaborDay_General/icon_02.png" style="border:none; display:block;">`

**Comment by adragin** [comment]
I work in email to. It's true? not working :(

Code:
```
<td>
	<img src="logo.png" width="" style="vertical-align:top;" alt="" />
</td>
```
keymap :
{ "keys": ["ctrl+u"], "caption": "Emmet: Update Image Size", "command": "emmet_update_image_size"}

And when I stay on image name and press ctrl+u nothing happens.

Traceback (most recent call last):
  File "C:\Sublime3\sublime_plugin.py", line 1088, in run_
    return self.run(edit)
  File "C:\Sublime3\Data\Installed Packages\Emmet.sublime-package\main.py", line 253, in run
  File "C:\Sublime3\Data\Installed Packages\Emmet.sublime-package\lib/update_image_size.py", line 17, in update_image_size
  File "C:\Sublime3\Data\Installed Packages\Emmet.sublime-package\lib/update_image_size.py", line 24, in update_image_size_html
AttributeError: 'module' object has no attribute 'tag'

**Comment by sergeche** [comment]
Oops, my bad, fixed in v2.1.9

**Comment by jakeanspach** [comment]
@sergeche Thank you!
